CONTRIBUTORS.md

```diff
@@ -11,6 +11,7 @@
 *   Eric Anderson
 *   Jason Kuster
```

Comment:
Repo contributors list including statistics (commits, lines of code) etc is available through [Github Insights](https://github.com/GoogleCloudPlatform/DataflowTemplates/graphs/contributors).  In general CONTRIBUTORS.md file include names  of primary contributors who has contributed major changes.  Please exclude the file from pull request.